04-onion-routing.md

```diff
@@ -235,10 +235,13 @@ parameters may lead to extraneous routing failure.
 
 ### `tlv_payload` payload format
 
-This is a more flexible format, which avoids the redundant `short_channel_id` field for the final node. 
+This is a more flexible format, which avoids the redundant `short_channel_id` field for the final node.
 
 1. tlvs: `tlv_payload`
 2. types:
+    1. type: 1 (`payment_secret`)
+    2. data:
+        * [`u16`:`payment_secret`]
```

Comment:
This is a 16 bit field!  You mean `16*bytes` I think.

Reply:
Woops definitely, thanks for spotting this